python/api.py

```diff
@@ -634,7 +647,11 @@ def to_data(self):
         obj["state"] = self.state                                              
         obj["timestamp"] = self.timestamp                                       
         obj["unique_id"] = self.unique_id                                        
-        obj["new_state"] = self.new_state                                        
+        obj["new_state"] = self.new_state
+        if len(self.active_ports) > 0:
```

Comment:
else

Reply:
this is to_data(), we don't need else case I think.